### DISCUSSION TITLE
add/remove columns in views

### DISCUSSION THREAD

**Discussion by Kyushin** [drakkan/sftpgo#642] [Ideas]
Hello !

I am very used to use Netbox and it have got a very usefull feature : add, rearrange and remove columns to fit our needs on views.
In my case, I would like to see user's description field in general user view.

Thanks !

**Comment by drakkan** [comment]
This is a good suggestion but js/html is not my preferred skill, I don't plan to work on this in the near future, sorry

**Comment by drakkan** [comment]
> Hello !
> 
> I am very used to use Netbox and it have got a very usefull feature : add, rearrange and remove columns to fit our needs on views. In my case, I would like to see user's description field in general user view.
> 
> Thanks !

Something basic is now supported

https://github.com/drakkan/sftpgo/commit/f536c64043d8475ed4980df8438fad7ce7b8a9dc
